web/themes/joinup/prototype/sass/components/_fieldset.sass

```diff
@@ -55,11 +62,12 @@
 
 .fieldset--border-bottom
   position: relative
-  padding-bottom: 18px
+  margin-bottom: 25px
+  padding-bottom: 27px
```

Comment:
Looks really `cheesy` - are you sure you really need these rules having all the other top/bottom spacing rules?

Reply:
It is only for the grey line fieldsets. It should be bulletproof, any issues with this?